command/untag_test.go

```diff
@@ -0,0 +1,11 @@
+package command
+
+import (
+	"testing"
+
+	"github.com/mitchellh/cli"
+)
+
+func TestTagCommand_implement(t *testing.T) {
```

Comment:
`TestTagCommand_implement` already exists. Please rename this to `TestUntagCommand_implement`